packages/mobile/src/store/offline-downloads/sagas/offlineQueueSagas/workers/downloadCollectionWorker.ts

```diff
@@ -58,18 +59,19 @@ export function* downloadCollectionWorker(collectionId: CollectionId) {
   )
   yield* put(startJob(queueItem))
 
-  const { jobResult, cancel, abort } = yield* race({
+  const { jobResult, cancel, abort, abortJob } = yield* race({
     jobResult: retryOfflineJob(
       MAX_RETRY_COUNT,
       1000,
       downloadCollectionAsync,
       collectionId
     ),
     abort: call(shouldAbortDownload, collectionId),
+    abortJob: call(shouldAbortJob),
```

Comment:
so would we not want this to be a part of "abort" as well?

Reply:
we take different action afterwards depending on whether it was cancelled or aborted, so I don't think that'll work

Reply:
yeah I was trying to think of the best way to combine them